pkg/spec/spec.go

```diff
@@ -173,6 +174,12 @@ type DeploymentSpecMod struct {
 	ext_v1beta1.DeploymentSpec `json:",inline"`
 }
 
+type JobSpecMod struct {
+	ControllerFields      `json:",inline"`
+	batch_v1.JobSpec      `json:",inline"`
+	ActiveDeadlineSeconds *int64 `json:"activeDeadlineSeconds,conflicting,omitempty"`
```

Comment:
It would be nice to have comment explaining why is `ActiveDeadlineSeconds` added here

Reply:
added